internal/quickstart/show_sdk_instructions.go

```diff
@@ -2,18 +2,21 @@ package quickstart
 
 import (
 	"fmt"
-
 	"github.com/charmbracelet/bubbles/help"
 	"github.com/charmbracelet/bubbles/key"
 	"github.com/charmbracelet/bubbles/spinner"
+	"github.com/charmbracelet/bubbles/viewport"
 	tea "github.com/charmbracelet/bubbletea"
 	"github.com/charmbracelet/glamour"
 	"github.com/charmbracelet/lipgloss"
-	"github.com/muesli/reflow/wordwrap"
-
 	"ldcli/internal/sdks"
 )
 
+const (
+	viewportWidth  = 80
+	viewportHeight = 30
+)
```

Comment:
arbitrary values I picked - we could adjust these based on the window size if we want?